src/questions/blockchain_basics.yaml

```diff
@@ -1,35 +1,1100 @@
+  type: SingleChoice
+  content: Blockchain can be stored as which of the following?
+  hint: NoHint
+  explanation: Blockchain details can be stored as a flat file and a database.
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: A flat file
+      key: A
+      order: 0
+    - content: A database
+      key: B
+      order: 1
+    - content: Both of the above
+      key: C
+      order: 2
+    - content: None of the above
+      key: D
+      order: 3
+- uuid: f807265d-3208-4a67-8d77-a25bc4bd2596
+  type: SingleChoice
+  content: A block in Blockchain cannot have more than one parent block?
+  hint: NoHint
+  explanation: A block in blockchain cannot have more than one parent because they have unique hash pointer attached to it. Hence , two blocks cannot point to the same block.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: True
+      key: A
+      order: 0
+    - content: False
+      key: B
+      order: 1    
+- uuid: 505ebda8-92ba-4922-bd0f-32ebd145b922
+  type: SingleChoice
+  content: Blocks in Blockchain as linked to each to each other .......
+  hint: Linking of the blockchain nodes
+  explanation: The blocks in Blockchain are attached to each other in the backward fashion joining to the previous node and so on.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Backward direction
+      key: A
+      order: 0
+    - content: Forward direction
+      key: B
+      order: 1
+    - content: Not connected
+      key: C
+      order: 2
+- uuid: a6ce806e-02dd-4195-b254-dfb5aa0092f3
+  type: MultipleChoice
+  content: Identify Features of Blockchain?
+  hint: Characteristics of Blockchain
+  explanation: Blockchain provides scalability , improved security to its users and they are immutable i.e. tamper proof.
+  answerKeys:
+    - A
+    - B
+    - C
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Scalability
+      key: A
+      order: 0
+    - content: Improved Security
+      key: B
+      order: 1
+    - content: Tamper Proof
+      key: C
+      order: 2
+    - content: Mutable
+      key: D
+      order: 3
+- uuid: 1f0ba670-bec3-4c83-aa0c-1ffb26177263
+  type: SingleChoice
+  content: What does P2P stand for?
+  hint: NoHint
+  explanation: P2P stands for peer-to-peer transaction which does not involve any centralised party.
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-blockchain"
+  difficultyLevel: "Low"
+  choices:
+    - content: Password to Password
+      key: A
+      order: 0
+    - content: Peer to Peer
+      key: B
+      order: 1
+    - content: Product to Product
+      key: C
+      order: 2
+    - content: Public key to Private Key
+      key: D
+      order: 3
+- uuid: 284d96ab-0336-4c2f-81cf-d6afe087d987
```

Comment:
I think we can remove this question `In Blockchain nodes are not in a distributed system.` its too basic

Reply:
Removed and added a new question